WcaOnRails/config/locales/en.yml

```diff
@@ -630,7 +630,7 @@ en:
       not_all_delegates: "are not all delegates"
       registration_close_after_open: "registration close must be after registration open"
       end_date_before_start: "End date cannot be before start date."
-      span_multiple_years: "Competition dates cannot span multiple years."
+      span_multiple_weeks: "Competition dates cannot span multiple weeks."
```

Comment:
Depending on your definition of a week, a 2 day competition that starts on a saturday and ends on a sunday techincally spans 2 weeks. I'd rephrase to something like "competition cannot last more than `%{max_competition_duration_days}`".